docs/user-guide/apis/java-api.md

```diff
@@ -0,0 +1,59 @@
+# Using the Java API
+
+As Gaffer is written in Java there is native support to allow use of all its
+public classes. Using Gaffer via the Java interface does differ from the rest
+API and `gafferpy` but is fully featured with extensive
+[Javadocs](https://gchq.github.io/Gaffer/overview-summary.html). However, you
+will of course need to be familiar with writing and running Java code in order
+to utilise this form of the API.
+
+## Querying a Graph
+
+Using Java to query a graph unlike the other APIs requires a reference to a
+`Graph` object that essentially represents a graph.
+
+With the other APIs you would connect directly to a running instance via the
+rest interface; however, to do this with Java you would need to configure a
+`Graph` object with a [proxy store](../../administration-guide/gaffer-stores/proxy-store.md).
+
+!!! example ""
+    The following example uses the `ProxyStore.Builder()` to configure a `Graph`
+    to connect to the required address (in this case
+    `http://localhost:8080/rest/latest`).
```

Comment:
Why the choice of `/rest/latest`? I think `/rest/` is the usual endpoint, or at least the ones used by the docker images?

Reply:
Think this might've been a typo from the v1 docs I'll change